docs/concepts/privacy.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Privacy
+sidebarDepth: 0
+description: Learn about user privacy and IPFS, the InterPlanetary File System.
+related:
+  'Article: IPFS Privacy (Textile)': https://medium.com/pinata/ipfs-privacy-711f4b72b2ea
+---
+
+# IPFS and privacy
+
+As you explore IPFS and learn about its core concepts, it's important to know that as a protocol for peer-to-peer data storage and delivery, IPFS in itself does not explicitly have a "privacy layer" built in. This is in line with key principles of protocol design, since different uses of IPFS over its (hopefully very long) life may call for different approaches to privacy. Explicitly implementing an approach to privacy within the IPFS core could "box in" future builders and implementers due to a lack of modularity, flexibility, and future-proofing.
```

Comment:
```suggestion
As you explore IPFS and learn about its core concepts, it's important to know that as a protocol for peer-to-peer data storage and delivery, IPFS in itself does not explicitly have a "privacy layer" built in. This is in line with key principles of protocol design, since different uses of IPFS over its life may call for different approaches to privacy. Explicitly implementing an approach to privacy within the IPFS core could "box in" future builders and implementers due to a lack of modularity, flexibility, and future-proofing.
```